Bhagyashri/js_ternary_evennumber/evenodd_number_ternary.js

```diff
@@ -0,0 +1,6 @@
+/*Find out even odd number.*/
+
+var num=10;
+
+(num%2==0)? document.write("Given Number is Even"): document.write("Given Number is Odd");
```

Comment:
var typeOfNumber = (num%2==0)? "Given Number is Even": "Given Number is Odd";
document.write(typeOfNumber);

Reply:
var num=17;
var typeofNumber=(num%2==0)?"Given Number is Even":"Given Number is Odd";
document.write(typeofNumber);

Tried this program but output is only even number.

Reply:
Your output is coming correctly I guess you are missing something or you are checking wrong program. 
Let's discuss over call if you are still unclear.